cluster-sync/sync.sh

```diff
@@ -28,21 +28,27 @@ if [ "${KUBEVIRT_PROVIDER}" != "external" ]; then
   registry=${IMAGE_REGISTRY:-localhost:$(_port registry)}
   DOCKER_PREFIX=${registry}
   MANIFEST_REGISTRY="registry:5000"
+  QUAY_NAMESPACE="none"
 fi
 
 # Need to set the DOCKER_PREFIX appropriately in the call to `make docker push`, otherwise make will just pass in the default `kubevirt`
+QUAY_NAMESPACE=$QUAY_NAMESPACE DOCKER_PREFIX=$MANIFEST_REGISTRY PULL_POLICY=$(getTestPullPolicy) make manifests
```

Comment:
Why the seperate QUAY_NAMESPACE?  We could just set DOCKER_PREFIX=quay.io/xxxxx

This also brings back my earlier comment that we should probably change DOCKER_PREFIX to REGISTRY_NAME or something more general.  This isn't your problem and shouldn't impact your patch but capturing the point here.

Reply:
QUAY_NAMESPACE is the namespace in quay where CDI OLM bundle is located. It is not necessarily equals to DOCKER_PREFIX

Reply:
I think Johns point is, that we either use Quay OR Docker but not both at the same time, so having two separate variables is sort of weird, it would make sense to have a 'REGISTRY_NAME' or something like instead of DOCKER_PREFIX and QUAY_NAMESPACE if they both serve the same purpose but for a different registry provider.

Reply:
> I think Johns point is, that we either use Quay OR Docker but not both at the same time, so having two separate variables is sort of weird, it would make sense to have a 'REGISTRY_NAME' or something like instead of DOCKER_PREFIX and QUAY_NAMESPACE if they both serve the same purpose but for a different registry provider.

But we do utilize both dockerhub and quay - we push all cdi images to repo in dockerhub and OLM bundle to quay

Reply:
Sorry, my point was as @awels mentioned why both, I get that we do push to both, but we don't push to both in the same build iteration.  So the idea I was getting at was use a single variable and set it appropriately for the build we're doing.

If you're doing parallel buld/push operations then it would make more sense to still use a single image registry variable but make it a list and just do build, then push to 'n' registries from the list.  We should probably be doing something like that anyway if we're not already rather than run a seperate build/release process for each registry.

Reply:
AS discussed we do need both DOCKER_PREFIX and QUAY_REPOSITORY variables in the same make manifests iteration. DOCKER_PREFIX for container images pushed to docker and QUAY related variables for olm manifests and bundle